src/fluree/db/db/json_ld.cljc

```diff
@@ -434,6 +435,8 @@
       (get @(:subclasses schema) class)
       (get-in schema [:pred class property])))
   (-p-prop [this property predicate] (graphdb-p-prop this property predicate))
+  (-expand-iri [this compact-iri] (expand-iri this compact-iri))
+  (-expand-iri [this compact-iri context] (expand-iri this compact-iri context))
```

Comment:
Just to check my understanding: the reason we're putting this on the record and not as a standalone function is so we have access to the default context?

Reply:
Correct. Expansion is typically handled automatically by the DB... but there is no way to know what the db will expand to without this.